python/cudf/cudf/tests/test_array_ufunc.py

```diff
@@ -23,11 +34,77 @@ def test_ufunc_cudf_series(np_ar_tup, func):
         assert_eq(expect, got.to_array())
 
 
+@pytest.mark.parametrize(
+    "func", [np.bitwise_and, np.bitwise_or, np.bitwise_xor],
+)
+def test_ufunc_cudf_series_bitwise(func):
+    x = np.random.randint(size=100, low=0, high=100)
+    y = np.random.randint(size=100, low=0, high=100)
+
+    s_1, s_2 = cudf.Series(x), cudf.Series(y)
+    expect = func(x, y)
+    got = func(s_1, s_2)
+    if np.isscalar(expect):
+        assert_eq(expect, got)
+    else:
+        assert_eq(expect, got.to_array())
+
+
 @pytest.mark.parametrize(
     "np_ar_tup", [(np.random.random(100), np.random.random(100))]
 )
 @pytest.mark.parametrize(
-    "func", [np.greater, np.less, np.less_equal],
+    "func",
+    [
+        np.subtract,
+        np.multiply,
+        np.floor_divide,
+        np.true_divide,
+        np.power,
+        np.remainder,
+        np.divide,
+        np.equal,
+        np.not_equal,
+        np.less,
+        np.less_equal,
+        np.greater,
+        np.greater_equal,
+    ],
+)
+def test_ufunc_cudf_series_cudf_dispatch(np_ar_tup, func):
```

Comment:
Usually we want to be able to set the seed for reproducibility. `np_ar_tup` should probably be updated to come from a data generating pytest fixture where the seed can be set, see a good example [here in the string tests.](https://github.com/rapidsai/cudf/blob/branch-0.18/python/cudf/cudf/tests/test_string.py#L55-L59).

Reply:
Resolved by https://github.com/rapidsai/cudf/pull/6973/commits/4be146433912fed09e094c81ea4881e7b7921df6